src/vuestic-theme/vuestic-components/va-table/VaTable.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <div class="va-table striped" :class="{'hoverable': clickable || hoverable}">
+    <slot name="header" />
+
+    <va-inner-loading :loading="loading">
+      <vuetable
+        ref="vuetable"
+        :api-mode="false"
+        :fields="fields"
+        :data="apiMode ? data : undefined"
+        :data-manager="apiMode ? undefined : dataManager"
+        :pagination-path="apiMode ? '' : 'pagination'"
+        :no-data-template="noDataLabel || $t('tables.noDataAvailable')"
+        :css="styles"
+        :row-class="rowClass"
+        @vuetable:row-clicked="rowClicked"
+      >
+        <!-- https://stackoverflow.com/questions/50891858/vue-how-to-pass-down-slots-inside-wrapper-component   -->
+        <template
+          v-for="slot in Object.keys($scopedSlots)"
+          :slot="slot"
+          slot-scope="scope"
+        >
+          <slot
+            :name="slot"
+            v-bind="scope"
+          />
+        </template>
+      </vuetable>
+
+      <div
+        v-if="!noPagination && paginationTotal > 1"
+        class="flex-center mt-3"
+      >
+        <va-pagination
+          v-model="currentPage"
+          :pages="paginationTotal"
+          :visible-pages="paginationVisible"
+          @input="inputPage"
+        />
+      </div>
+    </va-inner-loading>
+  </div>
+</template>
+
+<script>
+import { SpringSpinner } from 'epic-spinners'
+import Vuetable from 'vuetable-2/src/components/Vuetable'
+import VaPagination from '../va-pagination/VaPagination.vue'
+
+export default {
+  name: 'va-table',
+  components: {
+    SpringSpinner,
+    Vuetable,
+    VaPagination,
+  },
+  props: {
+    fields: {
+      type: Array,
+      required: true,
+    },
+    data: {
+      type: Array,
+      required: true,
+    },
+    perPage: {
+      type: Number,
+      default: 6,
+    },
+    apiMode: Boolean,
+    clickable: Boolean,
+    hoverable: Boolean,
+    noPagination: Boolean,
+    noDataLabel: {
+      type: String,
+      default: undefined,
+    },
+    rowClass: {
+      type: Function,
+      default: undefined,
+    },
+    totalPages: {
+      type: Number,
+      default: 0,
+    },
+    loading: Boolean,
+  },
+  data () {
+    return {
+      currentPage: 1,
+    }
+  },
+  computed: {
+    styles () {
+      return {
+        tableClass: 'va-table__vuetable' + (this.clickable ? ' clickable' : ''),
+        ascendingIcon: 'fa fa-caret-up',
+        descendingIcon: 'fa fa-caret-down',
+        renderIcon: classes => {
+          return '<span class="' + classes.join(' ') + '"></span>'
+        },
+      }
+    },
+    paginationTotal () {
+      return this.apiMode ? this.totalPages : Math.ceil(this.data.length / this.perPage)
+    },
+    paginationVisible () {
```

Comment:
Pretty sure this doesn't make any sense. Passing 4 here at all times should work the same.

Reply:
This was to limit visible pages. Resolved in the scope of: https://github.com/epicmaxco/vuestic-admin/pull/598